modules/persistent-storage-cinder-provisioning.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-cinder.adoc
+
+[id="persistent-storage-cinder-provisioning_{context}"]
+= Dynamic provisioning with Cinder
+
+Storage must exist in the underlying infrastructure before it can be mounted as a volume in {product-title}.
+
+.Prerequisites
+
+* {product-title} configured for OpenStack
+* Cinder volume ID
+* `*PersistentVolume*` API.
```

Comment:
This API is provided by OpenShift, and not something a customer would ever have to install or configure. It's more or less an implementation detail, and can be safely removed.